arcgis-ios-sdk-samples/Content Display Logic/Controllers/SourceCodeViewController.swift

```diff
@@ -41,29 +41,29 @@ class SourceCodeViewController: UIViewController, UIWebViewDelegate, UIAdaptiveP
         // Dispose of any resources that can be recreated.
     }
     
-    func loadHTMLPage(filename:String) {
+    func loadHTMLPage(_ filename:String) {
```

Comment:
I removed the `_` so its `loadHTMLPage(filename: String)`. Usage: `self.loadHTMLPage(filename: self.filenames[0])`